spec/models/o_auth_info_spec.rb

```diff
@@ -2,7 +2,17 @@
 
 describe OAuthInfo do
   let (:oauth) { described_class.new('info' => { 'name' => 'john doe' }) }
+  let (:oauth_invalid_chars) { described_class.new('info' => { 'name' => 'john.doe' }) }
 
+  context "should generate a username from the developer's oauth name" do
+  	it "should use the developer's oauth name if it is alphanumeric" do
```

Comment:
Can this indentation be converted to two spaces, per Ruby convention? Your text editor should be able to do this automatically, with some settings tweaks.